packages/SwingSet/src/lib/id.js

```diff
@@ -76,36 +69,6 @@ export function makeDeviceID(index) {
   return `d${Nat(index)}`;
```

Comment:
I think `makeVatID` and `makeDeviceID` should also be updated to ensure that they are never changed to create subtly invalid output such as "v0".

```js
export function makeVatID(index) {
  const vatID = `v${Nat(index)}`;
  insistVatID(vatID);
  return vatID;
}
```

```suggestion
  const deviceID = `d${Nat(index)}`;
  insistDeviceID(deviceID);
  return deviceID;
```

Reply:
Thats good suggestion. Added this check. Thanks